browser/src/App.js

```diff
@@ -1,33 +1,62 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
 import styles from './App.module.css';
 import { ContributeCard } from './components/contributeCard';
-import { useEffect, useState } from 'react';
 import { ENTROPY_ARRAY_MAX_SIZE } from './constants';
 
 const entropyArr = [];
 
 const CIRCUITS = process.env.REACT_APP_CIRCUITS.split(',').map(e => e.trim());
+const BACKEND_HOST = process.env.REACT_APP_BACKEND_HOST;
 
 function App() {
   const [mousePos, setMousePos] = useState({});
   const [doneCapturing, setDoneCapturing] = useState(false);
   const [entropy, setEntropy] = useState(0);
   const isMobile = window.innerWidth < 1024;
+  const [token, setToken] = useState('');
+  const [isButtonEnabled, enableButton] = useState(true);
```

Comment:
This one works 100%, but maybe we can keep the convrntion used across other files?

```js
const [buttonEnabled, setButtonEnabled] = setState(true);
```